src/test/java/coms/w4156/moviewishlist/serviceTests/WatchModeServiceTests.java

```diff
@@ -347,4 +338,259 @@ public void testGetFreeWithSubAllRentBuy() {
         Assertions.assertEquals(0, returnedSources.size());
     }
 
+    /************End Test getFreeWithSubscription function **************/
+
```

Comment:
These new tests exercise the new endpoints in symmetric ways as the old tests exercised the old endpoints